index.bs

```diff
@@ -2202,6 +2202,45 @@ During the above process, the user agent SHOULD show some UI to the user to guid
 authorizing an authenticator.
 </div>
 
+#### Registration API Exceptions
+
+[=[WRPS]=] can encounter the following {{DOMException|DOMExceptions}} from a call to {{CredentialsContainer/create()|navigator.credentials.create()}}. Some errors can have multiple reasons for why they happened, requiring the [=[WRPS]=] to infer the actual reason based on their use of WebAuthn:
+
+<dl dfn-type="argument" dfn-for="Registration API Exceptions">
+    :   <dfn>AbortError</dfn>
+    ::  The ceremony was cancelled via an {{AbortController}}
+        (see [[#sctn-abortoperation]] and [[#sctn-sample-aborting]] for more information.)
+
+    :   <dfn>ConstraintError</dfn>
+    ::  Either {{residentKey}} was set to "{{ResidentKeyRequirement/required}}" and no available authenticator supported resident keys,
+        or {{userVerification}} was set to "{{UserVerificationRequirement/required}}" and no available authenticator could perform [=user verification=].
+
+    :   <dfn>InvalidStateError</dfn>
+    ::  The authenticator used in the ceremony recognized an entry in {{excludeCredentials}}
```

Comment:
If the above suggestion of removing `dfn-for="Registration API Exceptions"` is accepted, then these `{{userVerification}}` and `{{excludeCredentials}}` references instead become ambiguous between the primary and JSON variants of the dictionary definitions, so we'd have to disambiguate these:

```suggestion
        or {{AuthenticatorSelectionCriteria/userVerification}} was set to "{{UserVerificationRequirement/required}}" and no available authenticator could perform [=user verification=].

    :   <dfn>InvalidStateError</dfn>
    ::  The authenticator used in the ceremony recognized an entry in {{PublicKeyCredentialCreationOptions/excludeCredentials}}
```